styles.css

```diff
@@ -0,0 +1,144 @@
+.container {
+  background: white;
+  border-bottom-left-radius: 50px;
+}
+
```

Comment:
Kindly update the styling of your headline section to better match the figma template🙏

Your design:
<img width="237" alt="image" src="https://user-images.githubusercontent.com/37116322/219500657-83bd3feb-5d64-4244-a02e-bf2e44b385f0.png">

Figma design:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/37116322/219500953-44306b77-0485-419c-86a9-4b18e70ea1e1.png">

A few things to note:
- Your font size, font weight and font family are off. Please make sure to check the figma template and make sure you are using the correct values for all those properties

**Please try to make your design match the figma template as much as possible**